src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/AdvSimd.PlatformNotSupported.cs

```diff
@@ -51,6 +51,30 @@ public new abstract class Arm64 : ArmBase.Arm64
             ///   A64: FSUB Vd.2D, Vn.2D, Vm.2D
             /// </summary>
             public static Vector128<double> Subtract(Vector128<double> left, Vector128<double> right) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// int8x8_t vrbit_s8 (int8x8_t a)
+            ///   A64: RBIT Vd.8B, Vn.8B
+            /// </summary>
```

Comment:
Can you please reorder the methods to keep consistency with other parts of this class:
* `Vector64<byte> ReverseElementBits(Vector64<byte> value)`
* `Vector64<sbyte> ReverseElementBits(Vector64<sbyte> value)`
* `Vector128<byte> ReverseElementBits(Vector128<byte> value)`
* `Vector128<sbyte> ReverseElementBits(Vector128<sbyte> value)`